src/DSharp.Compiler/Importer/MetadataImporter.cs

```diff
@@ -898,9 +897,9 @@ private void ImportScriptAssembly(MetadataSource mdSource, string assemblyPath,
         }
 
         private void ImportType(MetadataSource mdSource, TypeDefinition type, bool inScriptCoreAssembly,
-                                string scriptNamespace)
+                                string scriptNamespace, TypeSymbol outerType = null)
         {
-            if (type.IsPublic == false)
+            if (type.IsPublic == false && type.IsNestedPublic == false)
```

Comment:
`!` vs `==false` > `!`?

Reply:
haha subconsciously following the convention that was already there